lib/geokit/geocoders/mapbox.rb

```diff
@@ -9,7 +9,7 @@ class MapboxGeocoder < Geocoder
       private
 
       # Template method which does the reverse-geocode lookup.
-      def self.do_reverse_geocode(latlng)
+      def self.do_reverse_geocode(latlng, options = {})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used.